proto/v1/admin_api.proto

```diff
@@ -567,14 +566,11 @@ message OrderList {
 }
 
 message ListOrdersRequest {
-    string merchant_id = 1;
+    string order_id = 1;
+    repeated OrderStatus status = 2;
 
-    // Possible attributes on which expressions can be built on:
-    // - "id"
-    // - "status"
-    // - "userId"
-    // - "storeRef""
-    map<string, common.v1.ExpressionList> filter = 10;
+    string merchant_id = 10;
+    string store_id = 11;
```

Comment:
Isn't it actually better to have a generic filter of a generic ContentModel object? Although it's true, that we might use some other solution for that eventually.